index.js

```diff
@@ -117,22 +165,44 @@ var Umzug = module.exports = redefine.Class({
 
         return all.filter(function (migration) {
           return executedFiles.indexOf(migration.file) === -1;
-        }).sort(function (a, b) {
```

Comment:
Yep. I can't figure out any cases where `this._findMigrations()` resolves to a list of migrations already not in order. Otherwise, the sort should be in `_findMigrations()`?